e2e_tests/cli/test_import.py

```diff
@@ -311,3 +374,148 @@ def test_annotation_overwrite_warning(
+    )
+    result = run_cli_command(
+        f"darwin dataset import {local_dataset.name} darwin {expected_annotations_dir}"
+    )
+    assert_cli(result, 0)
+    base_slot = get_base_slot_of_item(config_values, local_dataset.id, item_idx=0)
+    with tempfile.TemporaryDirectory() as tmp_dir_str:
+        actual_annotations_dir = Path(tmp_dir_str)
+        export_and_download_annotations(
+            actual_annotations_dir, local_dataset, config_values
+        )
+        compare_annotations_export(
+            actual_annotations_dir, expected_annotations_dir, item_type, base_slot
+        )
+
+
+def test_import_annotations_to_multi_channel_item_without_slots_defined(
+    local_dataset: E2EDataset, config_values: ConfigValues
+) -> None:
+    """
+    Upload annotations to a multi-channel item without aligning each annotation to a
+    slot. All annotations should end up the base slot
+    """
+    item_type = "multi_channel"
+    local_dataset.register_read_only_items(config_values, item_type)
+    expected_annotations_dir = (
+        Path(__file__).parents[1]
+        / "data"
+        / "import"
+        / "multi_channel_annotations_without_slots_defined"
+    )
+    result = run_cli_command(
+        f"darwin dataset import {local_dataset.name} darwin {expected_annotations_dir}"
+    )
+    assert_cli(result, 0)
+    base_slot = get_base_slot_of_item(config_values, local_dataset.id, item_idx=0)
+    with tempfile.TemporaryDirectory() as tmp_dir_str:
+        actual_annotations_dir = Path(tmp_dir_str)
+        export_and_download_annotations(
+            actual_annotations_dir, local_dataset, config_values
+        )
+        compare_annotations_export(
+            actual_annotations_dir, expected_annotations_dir, item_type, base_slot
+        )
+
+
+def test_import_annotations_to_multi_channel_item_with_slots_defined(
+    local_dataset: E2EDataset, config_values: ConfigValues
+) -> None:
+    """
+    Upload annotations to a multi-channel item where each annotation is aligned with
+    the base slot. Each annotation should end up in the base slot
+    """
+    item_type = "multi_channel"
+    local_dataset.register_read_only_items(config_values, item_type="multi_channel")
+    expected_annotations_dir = (
+        Path(__file__).parents[1]
+        / "data"
+        / "import"
+        / "multi_channel_annotations_with_slots_defined"
+    )
+    result = run_cli_command(
+        f"darwin dataset import {local_dataset.name} darwin {expected_annotations_dir}"
+    )
+    assert_cli(result, 0)
+    base_slot = get_base_slot_of_item(config_values, local_dataset.id, item_idx=0)
+    with tempfile.TemporaryDirectory() as tmp_dir_str:
+        actual_annotations_dir = Path(tmp_dir_str)
+        export_and_download_annotations(
+            actual_annotations_dir, local_dataset, config_values
+        )
+        compare_annotations_export(
+            actual_annotations_dir, expected_annotations_dir, item_type, base_slot
+        )
```

Comment:
it looks like these methods are identical and only the `expected_annotations_dir` changes. I wonder if it can be useful to simplify. The rule of thumb is: if you repeat it 3 times than you can DRY the code. Up to you

Reply:
DRYing the code would make it way faster to read and understand